lib/bolt/node/ssh.rb

```diff
@@ -290,18 +290,32 @@ def _run_task(task, input_method, arguments)
           "PT_#{env}='#{val}'"
         end.join(' ')
       end
+     
+      use_sudo = @sudo || @run_as
```

Comment:
Why do we still have @sudo? It shouldn't be set anymore? If it is accomplishing something we should probably move this logic to a function.

Reply:
Ack, you're right.